link:
library/asyncio-task.html#asyncio.Task.cancelled

docs:


cancelled()Â¶
Return True if the Task is cancelled.
The Task is cancelled when the cancellation was requested with
cancel() and the wrapped coroutine propagated the
CancelledError exception thrown into it.
